src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/ChunkedEncodingWriteStream.cs

```diff
@@ -13,24 +13,23 @@ private sealed class ChunkedEncodingWriteStream : HttpContentWriteStream
         {
             private static readonly byte[] s_finalChunkBytes = { (byte)'0', (byte)'\r', (byte)'\n', (byte)'\r', (byte)'\n' };
 
-            public ChunkedEncodingWriteStream(HttpConnection connection, CancellationToken cancellationToken) :
-                base(connection, cancellationToken)
+            public ChunkedEncodingWriteStream(HttpConnection connection) : base(connection)
             {
             }
 
-            public override Task WriteAsync(byte[] buffer, int offset, int count, CancellationToken ignored)
+            public override Task WriteAsync(byte[] buffer, int offset, int count, CancellationToken ignored) // token ignored as it comes from SendAsync
```

Comment:
Should we assert that it is None?  Same below

Reply:
Yup, we can do that.